pkg/sriov/sriov.go

```diff
@@ -169,16 +152,9 @@ func (s *sriovManager) ReleaseVF(conf *sriovtypes.NetConf, podifName string, cid
 			return fmt.Errorf("failed to rename link %s to host name %s: %q", podifName, conf.OrigVfState.HostIFName, err)
 		}
 
-		// reset effective MAC address
-		if conf.MAC != "" {
-			hwaddr, err := net.ParseMAC(conf.OrigVfState.EffectiveMAC)
-			if err != nil {
-				return fmt.Errorf("failed to parse original effective MAC address %s: %v", conf.OrigVfState.EffectiveMAC, err)
-			}
-
-			if err = s.nLink.LinkSetHardwareAddr(linkObj, hwaddr); err != nil {
-				return fmt.Errorf("failed to restore original effective netlink MAC address %s: %v", hwaddr, err)
-			}
+		err = utils.SetVFEffectiveMAC(s.nLink, conf.OrigVfState.HostIFName, conf.OrigVfState.EffectiveMAC)
```

Comment:
can you keep original comment for consistency ?

Reply:
done